mewarpx/tests/test_coulomb_scattering.py

```diff
@@ -1,23 +1,20 @@
 """Tests for functionality in coulomb_scattering.py"""
 import numpy as np
 import pytest
+from pywarpx import callbacks, picmi
 import yt
 
+from mewarpx import coulomb_scattering, diags, emission, mespecies
+from mewarpx.mwxrun import mwxrun
+from mewarpx.poisson_solvers import DummyPoissonSolver
+from mewarpx.utils_store import mwxconstants, testing_util
+from mewarpx.utils_store import util
 from mewarpx.utils_store import util as mwxutil
```

Comment:
Should only have one of these...

Reply:
Weird that the `pre-commit` bot didn't catch that. Thanks!